include/z64item.h

```diff
@@ -320,28 +324,58 @@ typedef enum {
     /* 0x22 */ GI_QUIVER_30,
     /* 0x23 */ GI_QUIVER_40,
     /* 0x24 */ GI_QUIVER_50,
-    /* 0x25 */ GI_25 = 0x25,
-    /* 0x28 */ GI_NUTS_1 = 0x28,
-    /* 0x2A */ GI_NUTS_10 = 0x2A,
+    /* 0x25 */ GI_ARROW_FIRE,
+    /* 0x26 */ GI_ARROW_ICE,
+    /* 0x27 */ GI_ARROW_LIGHT,
+    /* 0x28 */ GI_NUTS_1,
+    /* 0x29 */ GI_NUTS_5,
+    /* 0x2A */ GI_NUTS_10,
+    /* 0x2E */ GI_BOMBCHUS_20 = 0x2E,
     /* 0x32 */ GI_SHIELD_HERO = 0x32,
     /* 0x33 */ GI_SHIELD_MIRROR,
-    /* 0x35 */ GI_MAGIC_BEANS = 0x35,
+    /* 0x34 */ GI_POWDER_KEG,
+    /* 0x35 */ GI_MAGIC_BEANS,
     /* 0x36 */ GI_BOMBCHUS_1,
-    /* 0x3A */ GI_BOMBCHUS_5 = 0x3A,
-    /* 0x3C */ GI_KEY_SMALL = 0x3C,
+    /* 0x37 */ GI_SWORD_KOKIRI,
+    /* 0x38 */ GI_SWORD_RAZOR,
+    /* 0x39 */ GI_SWORD_GILDED,
+    /* 0x3A */ GI_BOMBCHUS_5,
+    /* 0x3B */ GI_SWORD_GREAT_FAIRY,
+    /* 0x3C */ GI_KEY_SMALL,
     /* 0x3D */ GI_KEY_BOSS,
     /* 0x3E */ GI_MAP,
     /* 0x3F */ GI_COMPASS,
+    /* 0x41 */ GI_HOOKSHOT = 0x41,
+    /* 0x42 */ GI_LENS,
+    /* 0x43 */ GI_PICTO_BOX,
     /* 0x4C */ GI_OCARINA = 0x4C,
-    /* 0x50 */ GI_50 = 0x50,
-    /* 0x52 */ GI_SCALE_GOLD = 0x52, // Assumed, used in En_Fishing
-    /* 0x59 */ GI_BOTTLE_POTION_RED = 0x59,
+    /* 0x50 */ GI_BOMBERS_NOTEBOOK = 0x50,
+    /* 0x52 */ GI_SKULL_TOKEN = 0x52,
+    /* 0x55 */ GI_REMAINS_ODOLWA = 0x55,
+    /* 0x56 */ GI_REMAINS_GOHT,
+    /* 0x57 */ GI_REMAINS_GYORG,
+    /* 0x58 */ GI_REMAINS_TWINMOLD,
+    /* 0x59 */ GI_BOTTLE_POTION_RED, // ITEM_LONGSHOT
```

Comment:
```suggestion
    /* 0x59 */ GI_BOTTLE_POTION_RED, // ITEM_LONGSHOT
```
? Don't understand the comment

Reply:
There's some weird behavior going with this entry, all it's doing is noting the connection based on the get-item table in z_player: See row 90 to see that longshot and this get-item are connected:
https://docs.google.com/spreadsheets/d/1e46RySK9Zz4a4Yjt3lPiZ2jIqxZ9h_pMeD7W-_mbc7A/edit#gid=0

You can also check out this glitch mentioned before:
https://discord.com/channels/688807550715560050/688851976746041391/876032755664834571

I think we'll need `z_player.c` to figure out this one...

Reply:
Actually, I'll just post it here:

![image](https://user-images.githubusercontent.com/47598039/179746672-12145aa0-8fe2-4660-9b42-197257ee99f2.png)

Reply:
Ah, could you add a little clarifying comment about this, possibly even just a TODO to figure out the full connection?

Reply:
I mostly want to complain that the possessive of "thou" is "thine"...

Reply:
But there shouldn't be much visible here since it only exists on the first version and we're looking at the fourth.